client/ayon_maya/plugins/load/load_anim.py

```diff
@@ -0,0 +1,139 @@
+import os
+import json
+import maya.cmds as cmds
+import ayon_api
+from ayon_maya.api import plugin
+from ayon_core.pipeline.load.utils import get_representation_context
+from ayon_maya.plugins.load.load_reference import ReferenceLoader
+import logging
+import pymel.core as pm
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
+
+
+class AnimLoader(plugin.Loader):
+    """Load anim on character"""
+
+    product_types = {"animation"}
+    representations = {"anim"}
+
+    label = "Load Anim"
+    order = -5
+    icon = "code-fork"
+    color = "orange"
+
+    def load(self, context, name, namespace, data):
+        project_name = context['project']['name']
+        anim_file = context['representation']['attrib']['path']
+        self.log.info(f"anim_file: {anim_file}")
+        name_space = context['representation']['data']['context']['product']['name'].replace(
+            context['representation']['data']['context']['product']['type'], '')
+        if not cmds.namespace(exists=name_space):
+            assets = context['version']['data'].get("assets", [])
+            self.log.info(f"assets: {assets}")
```

Comment:
You can take reference from other loaders in `/load` folder to get the namespace
```suggestion
        folder_name = context["folder"]["name"]
        namespace = namespace or lib.unique_namespace(
            folder_name + "_",
            prefix="_" if folder_name[0].isdigit() else "",
            suffix="_",
        )

```